### ISSUE TITLE
Add tooltips to "Add" icons on space dashboard

### ISSUE THREAD

**Issue by ljelinkova** [openshiftio/openshift.io#4294]
I am sometimes confused what the "Add" button actually does (for example the Add button on Pipelines card adds new codebase). Is it possible to add tooltips to these icons?

![dashboard](https://user-images.githubusercontent.com/2078072/45029489-18cd2980-b049-11e8-80df-3fc29c417895.png)

**Comment by serenamarie125** [comment]
Agreed that this is a good idea.  That icon is used interchangeably for Create and Add, and we should explicitly state the appropriate action with context.

**Comment by serenamarie125** [comment]
@muruGanesan can you please take a look at this asap, and suggest the appropriate tooltips?

The tooltip should match the action, the modal/dialog title, and should also comply with the PatternFly [terminology guidelines](https://www.patternfly.org/styles/terminology-and-wording/):
- *Add* - Add an existing item to an existing list, group, view, or other container element
- *Create* - Create something new

**Comment by muruGanesan** [comment]
@serenamarie125 , I will provide an appropriate solution as per PF. 
[IMO: we might have defined this common behavior, if not we will add it to our OSIO-UXD wiki]

**Comment by serenamarie125** [comment]
+1 to adding to the OSIO-UXD wiki, but let's address the short term issues here asap by defining the content of these 3 tooltips.  Please cc Stacy & myself for quick review once available.  Thanks!

**Comment by muruGanesan** [comment]
Thanks, @AshishDurgude.
Here is the design proposal:
https://redhat.invisionapp.com/share/6RNVW22T2NM#/318163171_Tooltip_Standard_Scr2

**Comment by serenamarie125** [comment]
@muruGanesan the Invision provides a nice way to see how we access the tooltips.

But we also need to provide the content of these tooltips, as I mentioned above: 
> let's address the short term issues here asap by defining the content of these 3 tooltips. Please cc Stacy & myself for quick review once available. Thanks!

**Comment by muruGanesan** [comment]
@ljelinkova, the screenshot that you have provided is OLD dashboard view.
Here is the latest (implemented) screen from OSIO
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/26006817/45132068-27345600-b1ad-11e8-8aa4-79450c12d679.png">


But for your issue i.e. for yours & engineering reference: Whenever we add  (+) symbol for 'Add/Create", we need to provide tooltip as given below:

![tool-tip_old-reference](https://user-images.githubusercontent.com/26006817/45132158-77abb380-b1ad-11e8-9ebf-13e4048551a6.png)

**Comment by AshishDurgude** [comment]
Here is the invision link:
https://redhat.invisionapp.com/share/6RNVW22T2NM

**Comment by serenamarie125** [comment]
Agreed @ljelinkova! @muruGanesan please identify the appropriate tooltip for the Pipeline action as well.

**Comment by muruGanesan** [comment]
@serenamarie125, @ljelinkova,  there is no (+) symbol in pipeline card... Because, it would make any sense for the pipeline. 
This is an implementation issue.

Please refer to the design.
https://redhat.invisionapp.com/share/WQFA7QEUS#/screens/272611017
<img width="1527" alt="screen shot 2018-09-11 at 9 30 53 am" src="https://user-images.githubusercontent.com/26006817/45363397-6f84b700-b5a5-11e8-8361-4db8540c17ff.png">


I am removing the "UX label and assignee" since we have provided the solution.

**Comment by ljelinkova** [comment]
@muruGanesan Thanks for your help and I've added platform team to have a look.

**Comment by christianvogt** [comment]
Tooltip: `Add a Space`
![image](https://user-images.githubusercontent.com/14068621/49914113-135c5800-fe5e-11e8-8409-80800f7da8d3.png)

Tooltip: `Add a Codebase`
![image](https://user-images.githubusercontent.com/14068621/49914146-32f38080-fe5e-11e8-8099-285809a1a16b.png)

Tooltip: `Add a Work Item`
@muruGanesan please confirm as this button currently only brings the user to planner 0_o
![image](https://user-images.githubusercontent.com/14068621/49914312-c3ca5c00-fe5e-11e8-8112-cde88321ab3d.png)

@muruGanesan please confirm the tooltip text for the pipelines card:
![image](https://user-images.githubusercontent.com/14068621/49914178-528aa900-fe5e-11e8-8897-b1ae945ede77.png)

**Comment by muruGanesan** [comment]
@christianvogt,
I could see a lot of changes from the original 'space dashboard' design. 
I will recheck the design again & get back with an appropriate answer.



CC: @serenamarie125, @sunilmalagi, @AshishDurgude

**Comment by joshuawilson** [comment]
That is a good point, @Bumbilo please ensure this works on each feature flag level.

**Comment by muruGanesan** [comment]
I collected the latest space dashboard design from #4196. 
Here is the design file URL:
https://redhat.invisionapp.com/share/A5POLEI7TEG#/337627441_latest-SpaceDashboard


CC: @serenamarie125, @sunilmalagi, @AshishDurgude, @Veethika

**Comment by christianvogt** [comment]
@muruGanesan are these confirmed to be the final changes?

**Comment by muruGanesan** [comment]
@christianvogt, we will discuss and close in our stakeholder meeting (today) 19Dec18.

**Comment by muruGanesan** [comment]
@christianvogt,
As we discussed in today's meeting, I need to have an internal discussion and sync-up with another issue #4196 before I propose the final version. 

CC: @serenamarie125 @joshuawilson, @Veethika , @sunilmalagi , @AshishDurgude
